index.html

```diff
@@ -50,14 +53,18 @@
             </div>
 
             <div class="title">UWB Hacks the Cloud</div>
+            <div class="event">
+                <h4>April 17-19, 2020</h4>
```

Comment:
might still be good to include the kick-off time here, ending doesn't matter as much imo

Reply:
As long as the schedule is easily accessible I don't think it needs the start time here.